config/elasticsearch5.yml

```diff
@@ -0,0 +1,5 @@
+cluster.name: "docker-es5-cluster"
```

Comment:
Could these live under 'services/elasticsearchX', like we do for other services?